src/ray/raylet/task_dependency_manager.h

```diff
@@ -119,8 +138,14 @@ class TaskDependencyManager {
   /// operations to make the object available through object transfer or
   /// reconstruction.
   void HandleRemoteDependencyCanceled(const ObjectID &object_id);
+  void AcquireTaskLease(const TaskID &task_id);
 
   ObjectManagerInterface &object_manager_;
+  ReconstructionPolicyInterface &reconstruction_policy_;
+  boost::asio::io_service &io_service_;
+  const ClientID client_id_;
+  const int64_t initial_lease_period_ms_;
```

Comment:
Do we actually need this field? Or can we just get it directly from the config when we use it?

Reply:
I think it's nice to have it as a member so that we can test with different values than the config.